OpenRA.Mods.Common/Traits/Render/WithSpriteBody.cs

```diff
@@ -53,6 +53,12 @@ public class WithSpriteBody : PausableConditionalTrait<WithSpriteBodyInfo>, INot
 		readonly RenderSprites rs;
 		readonly Animation boundsAnimation;
 
+		// HACK: Needed to split fullness prefix of WithHarvestAnimation to WithHarvesterSpriteBody
```

Comment:
Wouldn't it make more sense to merge `WithHarvestAnimation` into `WithHarvesterSpriteBody` and avoid the hack?

Reply:
I'm not convinced, because a) that would bring us back to `WithHarvestAnimation` not being available to 'normal' sprite bodies, and b) actors which only need the harvesting anim and not the prefix logic would again be forced to carry the latter around as well, possibly breaking compatibility with other animation traits.
Besides, it's really only meant as a short-term, temporary hack that I still plan to replace with the priority logic before Next+1.